index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <style>
+        body{
+            background-color: rgba(104, 133, 80, 0.849);
+            background-image: url("https://wallpaperaccess.com/full/2041777.jpg");
+            text-emphasis-color: red;
+        }
+        #title{
+            text-align: center;
+            font-size: 40px;
+        }
+        #survey-form{
+            text-align: center;
+        }
+        #description{
+            text-align: center;
+            font-size: 20px;
+        }
+    </style>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>microverse survey form</title>
+</head>
+<body>
+    <h1 id="title" style = "color: red"> Survey form </h1>
+    <p id="description" style = "color: red"> This is a survey form </p>
+    <section>
+    <form id="survey-form" action="sumbit">
```

Comment:
I would suggest the use of indentation to improve the readability of the HTML file.